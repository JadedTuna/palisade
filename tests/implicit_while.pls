high secret = 34;
low guess = 0;
while(guess != secret) {
    guess = guess + 1;
}