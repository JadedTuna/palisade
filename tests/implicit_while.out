[1;35msecurity error: [0minsecure implicit flow - while loop using a high guard
   1 | high secret = 34;
   2 | low guess = 0;
   3 | [1;35mwhile[0m(guess != secret) {
       [1;35m[0m[1;35m^^^^^[0m
